commands/gamble.js

```diff
@@ -0,0 +1,202 @@
+const _ = require('lodash');
+
+const gamble = {
+    // currently only allow one game globally
+    currentGame: null,
```

Comment:
if we are tracking state I kind feel like this should just be a class.

Reply:
Yeah, perhaps. This was actually a last minute change to aid in testing. I went back and forth on all this.

In this specific case you brought up, a class doesn't really make sense. This object is a singleton, and so `gamble.currentGame` is a global reference to one game (i.e. currently, only one game can be going on at a time). So that's _not_ instance state where classes would make sense. It's a global state.